sdformat_mjcf_utils/sdformat_mjcf_utils/sdf_utils.py

```diff
@@ -61,6 +82,26 @@ def quat_to_euler_list(quat):
     return [math.degrees(val) for val in vec3d_to_list(quat.euler())]
 
 
+def get_pose_from_mjcf(element):
+    """
+    Get the pose from a MJCF element
+    :param mjcf.Element element: MJCF element to get the pose
+    :return: The newly created Pose3d
+    :rtype: ignition.math.Pose3d
+    """
+    pos = [0, 0, 0]
+    euler = [0, 0, 0]
+    try:
```

Comment:
Do we need the `try - except` if we're checking for `None`?

Reply:
Yes, the worldbody doesn't have these elements, using the `try - except` allows to use the same function for `worldbody` and `bodies`.

Reply:
Got it. Thanks!